python/ray/tune/suggest/dragonfly.py

```diff
@@ -250,6 +253,13 @@ def _setup_dragonfly(self):
         self.init_dragonfly()
 
     def init_dragonfly(self):
+        if self._points_to_evaluate:
+            self._points_to_evaluate = [[
```

Comment:
should you save this to a separate name?

or, is it possible that init_dragonfly is called twice (perhaps in save/restore mode)?

Reply:
`init_dragonfly` shouldn't be called twice, but I'll use a local variable here for the transformation so we don't ever run into errors. The points are then kept in `self._initial_points` anyway.